tests/everflow/test_everflow_testbed.py

```diff
@@ -646,6 +646,137 @@ def test_everflow_dscp_with_policer(
             everflow_utils.remove_route(everflow_dut, self.DEFAULT_DST_IP + "/32", default_traffic_peer_ip,
                                         setup_info[default_tarffic_port_type]["remote_namespace"])
 
+    def test_everflow_frwd_with_bkg_trf(self,
+                                        setup_info, # noqa F811
+                                        setup_mirror_session,
+                                        dest_port_type, ptfadapter, tbinfo,
+                                        skip_traffic_test # noqa F811
+                                        ):
+        """
+        Verify basic forwarding scenarios for the Everflow feature with background traffic.
+        Background Traffic PKT1 IP in IP with same ports & macs but with dummy ips
+        Background Traffic PKT2 Packer with same ports & macs but with dummy ips
+        """
+        everflow_dut = setup_info[dest_port_type]['everflow_dut']
+        remote_dut = setup_info[dest_port_type]['remote_dut']
+        remote_dut.shell(remote_dut.get_vtysh_cmd_for_namespace(
+            "vtysh -c \"configure terminal\" -c \"no ip nht resolve-via-default\"",
+            setup_info[dest_port_type]["remote_namespace"]))
+
+        # Add a route to the mirror session destination IP
+        tx_port = setup_info[dest_port_type]["dest_port"][0]
+        peer_ip = everflow_utils.get_neighbor_info(remote_dut, tx_port, tbinfo)
+        everflow_utils.add_route(remote_dut, setup_mirror_session["session_prefixes"][0], peer_ip,
+                                 setup_info[dest_port_type]["remote_namespace"])
+
+        time.sleep(15)
+
+        # Verify that mirrored traffic is sent along the route we installed
+        rx_port_ptf_id = setup_info[dest_port_type]["src_port_ptf_id"]
+        tx_port_ptf_id = setup_info[dest_port_type]["dest_port_ptf_id"][0]
+
+        stop_thread = threading.Event()
+
+        def background_traffic():
+
+            router_mac = setup_info[dest_port_type]["ingress_router_mac"]
+            inner_pkt2 = self._base_tcp_packet(ptfadapter, setup_info, router_mac, src_ip="1.0.0.10", dst_ip="4.4.4.4")
+            bkg_trf1 = testutils.simple_ipv4ip_packet(ip_src="30.0.0.10",
+                                                      eth_dst=router_mac,
+                                                      ip_dst="50.0.2.2",
+                                                      eth_src=ptfadapter.dataplane.get_mac(0, 0),
+                                                      inner_frame=inner_pkt2[scapy.IP])
+            bkg_trf2 = self._base_tcp_packet(ptfadapter, setup_info, router_mac, src_ip="30.0.0.10", dst_ip="50.0.2.2")
+            logger.debug("Background Traffic Started")
+            while not stop_thread.is_set():
+                testutils.send(ptfadapter, rx_port_ptf_id, bkg_trf1)
+                testutils.send(ptfadapter, rx_port_ptf_id, bkg_trf2)
```

Comment:
let's add verification in test to make sure background packet egress out.

Reply:
add v4 and v6 traffic?

Reply:
@abdosi 
Hello,
Verification of each packet of the background traffic is creating issue while polling the device for packet for mirrored traffic.
In order to avoid this situation can we verify few packet of background traffic at the start and at end and run the background traffic without verification during the mirrored traffic processing  ?